src/api/integrations/APIM/index.ts

```diff
@@ -115,6 +121,54 @@ const APIM = {
       throw new Error(`Calling APIM - currencies exchange ${error}`);
     }
   },
+  /**
+   * sendEmail
+   * Send an email via APIM
+   * @param {String} templateId: Template ID
+   * @param {String} sendToEmailAddress: Email recipient
+   * @param {Object} personalisation: Custom variables for the email template
+   * @returns {ApimSendEmailHelperResponse}
+   */
+  sendEmail: async (templateId: string, sendToEmailAddress: string, personalisation: object): Promise<ApimSendEmailHelperResponse> => {
+    try {
+      console.info('Calling APIM - send email - templateId %s', templateId);
+
+      const response: ApimSendEmailResponse = await axios({
+        method: 'post',
+        url: `${APIM_MDM_URL}${APIM_MDM.EMAIL}`,
+        headers: {
+          'Content-Type': 'application/json',
+          [String(APIM_MDM_KEY)]: APIM_MDM_VALUE,
+          govUkNotifyKey: GOV_NOTIFY_API_KEY,
+        },
+        data: {
+          templateId,
+          sendToEmailAddress,
+          personalisation,
+        },
+        validateStatus(status) {
+          const acceptableStatus = [201];
+          return acceptableStatus.includes(status);
+        },
+      });
+
+      if (response.data && response.status === 201) {
```

Comment:
```suggestion
      if (response?.data && response?.status === 201) {
```

Reply:
Suggest using HttpStatusCode not sure if this is something EXIP uses widely?

Reply:
wondering if you can use `validateStatus` instead of explicit status check to `201` since same operation is being performed when calling `validateStatus`.

Reply:
It doesn't seem wise to remove the specific status check.

If APIM returns a 500 status, there will be no `response.status` and instead a `response.statusCode`. 

In addition, there could or could not be a `data` object.

Therefore I think it should remain as is, as a fail safe. Especially considering that this call is absolutely essential for the entire EXIPS service.

There could be many different combinations of responses. The current specific status check - with the check to ensure a data object is in the response, seems to be a failsafe check for a successful response.

Reply:
We don't have `HttpStatusCode` in EXIP unfortunately - would be introduce to have in the future.

EXIPS only has ~10 instances of status specific checks.

Reply:
Apologies for the confusion, I meant it will be wise to do following to keep the code DRY?

```ts
const isValidResponse = response.data && validateStatus(response.status);

if(isValidResponse) {...
```

The function is Axios call can be centralised.

```ts
 const validateStatus = (status) => {
          const acceptableStatus = [201];
          return acceptableStatus.includes(status);
        },
```